<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicTestsClass</class>
 <widget class="QMainWindow" name="BasicTestsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BasicTests</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="TzArrayTestTab" name="ArrayTestTab">
       <attribute name="title">
        <string>ArrayTestTab</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="Array_start_pushButton">
            <property name="text">
             <string>开始</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Array_end_pushButton">
            <property name="text">
             <string>结束</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QProgressBar" name="ArrayTestProgressBar">
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="invertedAppearance">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="Array_textEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="BitFlagsTestTab">
       <attribute name="title">
        <string>BitFlagsTestTab</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QComboBox" name="comboBox_bit_test">
          <item>
           <property name="text">
            <string>用例1 —— 与操作</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>用例2 —— 或操作</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>用例3 —— +操作</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>用例4 —— -操作</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>用例5 —— 非操作</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>用例6 —— 数据设置与验证</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>用例7 —— 综合操作验证</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar_bit_test">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_bit_test"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ToolButtonTestTab">
       <attribute name="title">
        <string>ToolButton_Test</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QToolButton" name="toolButton_test">
          <property name="text">
           <string>...</string>
          </property>
          <property name="popupMode">
           <enum>QToolButton::DelayedPopup</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TzArrayTestTab</class>
   <extends>QWidget</extends>
   <header>basic_array_test_tab_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="BasicTests.qrc"/>
 </resources>
 <connections/>
</ui>
